README.md

```diff
@@ -1,122 +1,598 @@
-Sheer
-===================
+# Sheer
+
+Sheer is a tool for [loading arbitrary content](#indexing) into [Elasticsearch](http://www.elasticsearch.org) and [serving that content](#serving) on the web using [Jinja2 templates](http://jinja.pocoo.org/). 
+
+If you're not familiar with Elasticsearch, it is highly recommended that you read the Elasticsearch Definitive Guide's [Finding Your Feet](http://www.elasticsearch.org/guide/en/elasticsearch/guide/current/_finding_your_feet.html).
+
+* [Requirements](#requirements)
+* [Installation](#installation)
+* [General Usage](#general-usage)
+* [Testing](#testing)
+* [Quick Start](#quick-start)
+* [Indexing](#indexing)
+    * [Reindexing](#reindexing)
+    * [Partial Indexing](#partial-indexing)
+	* [Sheer Index Settings](#sheer-index-settings)
+	* [Content Processors](#content-processors)
+	* [Mappings](#mappings)
+* [Serving](#serving)
+	* [Templates](#templates)
+		* [Context](#context)
+			* [`selected_filters_for_field`](#selected_filters_for_fieldfieldname)
+			* [`is_filter_selected`](#is_filter_selectedfieldname-value)
+			* [`queries`](#queries)
+			* [`more_like_this`](#more_like_thishit)
+			* [`get_document`](#get_documentdoctype-docid)
+	* [Elasticsearch Lookup URLs](#elasticsearch-lookup-urls)
+	* [Blueprints](#blueprints)
+	* [Query API](#query-api)
+		* [`QueryFinder`](#queryfinder)
+		* [`Query`](#query)
+		* [`QueryResult`](#queryresult)
+		* [`QueryHit`](#queryhit)
+- [Licensing](#licensing)
+
+
+## Requirements
+
+Sheer is a Python application that requires:
+
+- [Elasticsearch](http://www.elasticsearch.org/)
+- [Flask](http://flask.pocoo.org/)
+
+Recommended for installing and running Sheer:
+
+- [virtualenv](https://virtualenv.pypa.io/en/latest/)
+- [virtualenvwrapper](https://virtualenvwrapper.readthedocs.org)
+
+Running tests requires:
+
+- [nose](https://nose.readthedocs.org/en/latest/)
+- [mock](http://www.voidspace.org.uk/python/mock/)
+
+## Installation
+
+To install Sheer you will first need to install 
+[Elasticsearch](http://www.elasticsearch.org/). This can be acomplished 
+a number of ways, many of which are detailed in 
+[in the Elasticsearch documentation](http://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html). 
+On Mac OS X it can be installed using [homebrew](http://brew.sh/):
 
-Sheer provides:
+```
+brew install elasticsearch
+```
+
+There are also [Elasticsearch apt and Yum repositories](http://www.elastic.co/guide/en/elasticsearch/reference/current/setup-repositories.html).
```

Comment:
Ok, I've added a bit on how to ensure Elasticsearch runs. I tried to make the text you have a bit more generic and less applicable to our specific CFPB machines (see other threads on this point). I also think it's unnecessary to include the `brew info elasticsearch` since we're giving the user same info that homebrew provides, not telling to check what homebrew says to do. They can (or will be able to) check back in the Sheer README.